rakefile.rb

```diff
@@ -95,6 +101,9 @@
 desc "Execute default tasks"
 task :default => [ :vars, :gitlink, :unit, :integ, :spec, :pack ]
 
+desc "Execute default tasks plus .NET Core tasks"

```

Comment:
It's not really ".NET Core", it's ".NET Platform Standard".

".NET Core" is just one of the platforms that our ".NET Platform Standard" DLL supports.